_readme-install-epinowcast.Rmd

```diff
@@ -0,0 +1,23 @@
+You can install the latest released version using the normal `R` function, though you need to point to `r-universe` instead of CRAN:
```

Comment:
Nice idea. I think we want this in a folder - maybe in `man/chunks` (?) and to be in the build ignore?